specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-15-preview/machineLearningServices.json

```
@@ -0,0 +1,5532 @@
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Get",
+        "description": "Get the detail of a linked workspace.",
+        "x-ms-examples": {
+          "GetLinkedWorkspace": {
+            "$ref": "./examples/getLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Delete",
+        "description": "Delete a linked workspace.",
+        "x-ms-examples": {
+          "DeleteLinkedWorkspace": {
+            "$ref": "./examples/deleteLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully deleted the linked workspace."
+          },
+          "204": {
+            "description": "Specific linked workspace not found."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/services": {
+      "get": {
+        "tags": [
+          "MachineLearningServices"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "operationId": "MachineLearningService_ListByWorkspace",
```

Usually an operationId should have a plural noun as its receiver, for instance
```suggestion
        "operationId": "MachineLearningServices_ListByWorkspace",
```